frameworks/Java/beyondj/setup.sh

```diff
@@ -0,0 +1,16 @@
+WORKING_DIR=beyondj-launcher/deploy
+#FILE=beyondj-launcher/deploy/beyondj.jar
+
+if [ ! -d "$WORKING_DIR" ]; then
+        mkdir $WORKING_DIR
+fi
+
+#if [ ! -f "$FILE" ]; then
+        cd beyondj-launcher/deploy/
+        wget http://beyondj.com/beyondj.jar
```

Comment:
There is no problem with providing the SNAPSHOT version. I'll add version and update